portfolio/src/main/webapp/comments/CommentFacade.js

```diff
@@ -1,9 +1,34 @@
 class CommentFacade {
-    constructor(commentFetcher, commentRenderer) {
+    constructor(commentFetcher, commentRenderer, commentFormRenderer) {
         this.commentFetcher = commentFetcher;
         this.commentRenderer = commentRenderer;
+        this.commentFormRenderer = commentFormRenderer;
+
+        this.state = {
+            user: false,
+            comments: []
+        };
 
         this.refreshComments();
+        this.userUpdate(this.state.user);
+    }
+
+    userUpdate(email) {
+        const user = (email != null); // boolean
```

Comment:
If this is a boolean I would recommend a variable name like hasUser

Reply:
Resolved